open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/README.md

```diff
@@ -45,12 +45,7 @@ with security enforcement points.
 
    * These connectors will become Security Integration Connectors for the 
      [Security Integrator Open Metadata Integration Service (OMIS)](../../../integration-services/security-integrator)
```

Comment:
Should the security officers and security sync connectors beings removed

Reply:
thanks, will remove